esp/public/media/scripts/ajaxschedulingmodule/ESP/Sections.js

```diff
@@ -47,6 +48,9 @@ function Sections(sections_data, section_details_data, teacher_data, scheduleAss
         }.bind(this));
         return result;
     }.bind(this);
+    this.filter.classHideUnapproved.valid = function(a) {
+        return a.status === 10;
```

Comment:
status > 0 is used in a couple of places, notably ClassSubject.isAccepted():
https://github.com/learning-unlimited/ESP-Website/blob/main/esp/esp/program/models/class_.py#L1759